tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -54,29 +58,105 @@ struct SynthOrdered {
     }
 };
 
-template <class T>
-inline constexpr bool is_pair = false;
-template <class A, class B>
-inline constexpr bool is_pair<std::pair<A, B>> = true; // TRANSITION, std::pair spaceship not yet implemented
+struct OrderedChar {
+    OrderedChar() = default;
+    OrderedChar(const char c) : c(c){};
 
-template <class Container>
-void ordered_containers_test(const Container& smaller, const Container& smaller_equal, const Container& larger) {
+    OrderedChar& operator=(const char& other) {
+        this->c = other;
+        return *this;
+    }
+
+    operator char() {

```

Comment:
Should this be `const`?

Reply:
sure

Reply:
And `nodiscard`?

Reply:
We haven't been `_NODISCARD`ing conversion operators: the spec says that `[[nodiscard]]` produces warnings only for discarded function call expressions and people rarely call `meow.operator char();` directly, discarded or not (https://godbolt.org/z/4sh4or).